<% if error_contents.errors.any? %>
  <div id="error_explanation">
    <div class="alert alert-error">
      入力に <%= error_contents.errors.count %> 個のエラーがあります。
      <br>
      <ul>
      <% error_contents.errors.full_messages.each do |msg| %>
        <li>* <%= msg %></li>
      <% end %>
      </ul>
    </div>
  </div>
<% end %>
